frontend/src/v5/helpers/viewpoint.helpers.ts

```diff
@@ -160,19 +168,23 @@ export const meshObjectsToV5GroupNode = (objects) => objects.map((obj) => ({
 	_ids: obj.mesh_ids,
 }));
 
-export const toColorAndTransparencyDicts = (overrides: GroupOverride[]): OverridesDicts => {
-	const toMeshDictionary = (objects: V4GroupObjects, color: string, opacity: number): OverridesDicts => objects.shared_ids.reduce((dict, id) => {
-		if (color !== undefined) {
+export const toGroupPropertiesDicts = (overrides: GroupOverride[]): OverridesDicts => {
+	const toMeshDictionary = (objects: V4GroupObjects, color: string, opacity: number): OverridesDicts => 
+		objects.shared_ids.reduce((dict, id) => {
+			if (color !== undefined) {
 			// eslint-disable-next-line no-param-reassign
-			dict.overrides[id] = color;
-		}
+				dict.overrides[id] = color;
+			} else {
+				dict.transparencies[id] = 0;
```

Comment:
If the color is undefined, isn't the opacity always defined?

Reply:
I think this is a leftover